genai/image_generation/imggen_mmflash_edit_img_with_txt_img.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def generate_content() -> str:
+    # [START googlegenaisdk_imggen_mmflash_edit_img_with_txt_img]
+    from google import genai
+    from google.genai.types import GenerateContentConfig
+    from PIL import Image
+    from io import BytesIO
+
+    client = genai.Client()
+
+    # Using an image of Eiffel tower, with fireworks in the background.
+    image = Image.open("example-image.png")
+
+    response = client.models.generate_content(
+        model="gemini-2.0-flash-preview-image-generation",
+        contents=[image, "Edit this image to make it look like a cartoon."],
+        config=GenerateContentConfig(response_modalities=["Text", "Image"]),
```

Comment:
```suggestion
        config=GenerateContentConfig(response_modalities=["TEXT", "IMAGE"]),
```

I think the general procedure is to use uppercase. Has the SDK added an ENUM for these values yet?

Reply:
Updated to use `Modality.TEXT, Modality.IMAGE`